README.md

```diff
@@ -19,24 +19,17 @@ Dependencies
 
 In order to build GXUI on linux, you will need the following packages installed:
 
-    sudo apt-get install libxi-dev libxcursor-dev libxrandr-dev libxinerama-dev mesa-common-dev libgl1-mesa-dev libxxf86vm-dev
+    sudo apt-get install libxi-dev libxcursor-dev libxrandr-dev libxinerama-dev mesa-common-dev libgles2-mesa-dev libxxf86vm-dev
```

Comment:
Resolved via goxjs/gl#9 and https://github.com/google/gxui/commit/b022e995a0719c25fd40537792ac4ea0338ef204.